gef.py

```diff
@@ -4619,15 +4620,9 @@ def get_pcustom_abspath(self):
 
 
     def get_pcustom_filepath_for_structure(self, structure_name):
-        structure_files = self.enumerate_structures()
-        fpath = None
-        for fname in structure_files:
-            if structure_name in structure_files[fname]:
-                fpath = fname
-                break
-        if not fpath:
-            raise FileNotFoundError("no file for structure '{}'".format(structure_name))
-        return fpath
+        root = self.get_pcustom_abspath()
```

Comment:
This doesn't work as what we want if there are multiple structures that we want visible from `pcustom` in the file.

Reply:
I don't get it. Maybe this function should be named get_pcustom_filepath_for_module? Because that is how it is used on https://github.com/hugsy/gef/blob/dev/gef.py#L4760

Reply:
Yes, it is used only to get path for module and not for structures.